README.md

```diff
@@ -1,7 +1,5 @@
 # Cloud Application Framework
 
-![logo](logo.png)
```

Comment:
No more logo? 😢

Reply:
Haha I dunno, it was kind garish, I liked the bit in the middle but the bright blue background, I dunno! I might try and resuscitate it with the bits I actually like